File: CSharp/rate.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
	Scanner dur = new Scanner(System.in);
	double rate = 5_000;

	System.out.println("Enter Shift duration");

	int hours = dur.nextInt();

	
	if(hours < 0 || hours > 84){
	    System.out.println("Out of range");
	}
	else if(hours > 40){
	    double pay;
	    pay = 1.5 * hours * rate;
	    System.out.println("₦" + pay);
	}
	else if(hours <= 40){
	    double pay;
	    pay = hours * rate;
	    System.out.println("₦" + pay);
	}
	dur.close();
    }
}

File: CSharp/rate.cs
using System;

namespace rate{
    class wage{
	static void Main(string[] args){
	    double rate = 5000;

	    Console.WriteLine("Enter shift duration");
	    
	    string time = Console.ReadLine();
	    int hours = Convert.ToInt32(time);

	    if(hours < 0 || hours > 84){
		Console.WriteLine("Out of range");
	    }
	    else if(hours > 40){
		double pay;
		pay = 1.5 * hours * rate;
		Console.WriteLine("₦" + pay);
	    }
	    else if(hours <= 40){
		double pay;
		pay = hours * rate;
		Console.WriteLine("₦" + pay);
	    }
	}
    }
}
